sample-app/src/components/AppliedFilters.tsx

```diff
@@ -75,7 +75,12 @@ function RemovableFilter({ filter }: {filter: DisplayableFilter }): JSX.Element
   }
 
   const onRemoveStaticFilterOption = () => {
-    document.getElementById(`${filter.filter.fieldId + "_" + filter.filter.value}`)?.click();
+    if (!filter.filterCollectionId) {
```

Comment:
do we need this console.error if answers-headless also logs an error for this?

Reply:
filterCollectionId can't be required for DisplayableFilter since DisplayableFilter can also be of type Facet or NLP filters, which doesn't have filterCollectionId. Typescript would yell that `filter.filterCollectionId` can be undefined, which is not allow for function setFilterOption. So a check would be needed here anyway. I think it's ok to keep? Or I can remove the console.error line to avoid duplicate message